src/applications/mhv/secure-messaging/tests/e2e/pages/PatientComposePage.js

```diff
@@ -44,14 +44,18 @@ class PatientComposePage {
   };
 
   verifySendMessageConfirmationMessage = () => {
-    cy.get('.vads-u-margin-bottom--1').should(
```

Comment:
@fazilqa @haniasma I would suggest avoiding relying on classes as `vads-u-margin--***` and similar as this can get you anywhere easily